lib/rouge/lexers/raml.rb

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*- #
+
+module Rouge
+  module Lexers
+    load_lexer 'yaml.rb'
```

Comment:
This is not necessary. `YAML` is already loaded. See examples for other Lexers that subclass - https://github.com/jneen/rouge/blob/master/lib/rouge/lexers/hylang.rb. I tested this locally to be sure it works without and it does.